enhancements/substitutes-for.md

```diff
@@ -0,0 +1,386 @@
+---
+title: Substitutes-For-Graph-Replacements
+authors:
+  - "@bparees"
+reviewers:
+  - "@ecordell"
+  - "@kevinrizza"
+  - "@lgalleti"
+approvers:
+  - TBD
+creation-date: 2020-12-17
+last-updated: 2020-12-17
+status: provisional
+---
+
+# Bundle Substitution
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions [optional]
+
+1. Can we get the operator pipeline using a single opm binary that is always the latest
```

Comment:
The short answer is that there will be tradeoffs as long as opm is part of the openshift product and not a separate errata (because there is a period of time where the latest opm can't be used the publish content to public registry.redhat.io). The long answer is [probably this](https://hackmd.io/8pb60u3KQtiamveuyBYasA?view)

(this is the type of concern that I would expect to be more in an openshift/enhancement though).